lib/proxy/log_buffer/decorator.rb

```diff
@@ -10,17 +10,15 @@ def initialize(logger, buffer = Proxy::LogBuffer::Buffer.instance)
 
     def add(severity, message = nil, progname = nil, backtrace = nil, a_module = 'core', &block)
```

Comment:
What @dLobatog said. Unless you explicitly call ```block.call```, no need to include block as a parameter in method's argument list.